<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertNewItem</class>
 <widget class="QDialog" name="InsertNewItem">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
     QWidget {
     background-color: rgb(191,193,194);
     border-color: black;
     }

     /*	Push button*/
     QPushButton {
     min-height: 20px;
     border: 1px solid gray;
     border-radius: 3px;
     background-color: lightgray;
     selection-background-color: gray;
     }

     QPushButton:hover {
     background-color: gray;
     border-radius: 3px;
     }

     /*Menu bar*/
     QMenuBar {
     background-color:rgb(105,105,105);
     spacing: 0px;
     }

     QMenuBar::item {
     background-color: transparent;
     }

     QMenu {
     background-color:rgb(105,105,105);
     border: 1px solid black;
     }

     /*Combo box*/
     QComboBox {
     min-height: 19px;
     background-color: lightgray;
     selection-background-color: gray;
     padding-left: 4px;
     border: 1px solid gray;
     }

     QComboBox::down-arrow {
     image: url(../../../Resources/Icons/arrow_down.png);
     width: 10px;
     height: 10px;
     }

     QComboBox::drop-down {
     border: 0px;
     subcontrol-position: top right;
     width: 15px;
     }

     QComboBox QAbstractItemView {
     border: 1px solid black;
     background-color: lightgray;
     selection-background-color: gray;
     }


     QMenuBar::item:selected, QMenu::item:selected,QListWidget::item:selected, QComboBox:hover
     {
     background-color: gray;
     }


     /*Tool button*/
     QToolButton {
     max-width: 20px;
     max-height: 20px;
     border: 0px solid gray;
     }

     /*Line edit*/
     QLineEdit {
     border: 1px solid gray;
     border-radius: 3px;
     background-color: lightgray;
     selection-background-color: darkgray;
     }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="m_pFrameTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>290</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="m_pLabelTitle">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Insert New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_pToolButtonClose">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Resources/Icons/close-icon.png</normaloff>../../../Resources/Icons/close-icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_pVerticalFrameListWidgets">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QListWidget" name="m_pListWidgetGroup">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget::item:hover,QListWidget::item:selected{
	background-color: rgb(105, 105, 105);
	color:white;
}

QListWidget{
	background-color: rgb(191,193,194);
	border-radius: 3px;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Equipment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Geometric Shape</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Product</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_pArrow">
        <property name="maximumSize">
         <size>
          <width>330</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../Resources/Icons/arrow_down.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="m_pListWidgetItem">
        <property name="styleSheet">
         <string notr="true">QListWidget::item:hover,QListWidget::item:selected{
	background-color: rgb(105, 105, 105);
	color:white;
}

QListWidget{
	background-color: rgb(191,193,194);
	border-radius: 3px;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_pFrameButtonBox">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_pPushButtonInsert">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	min-height: 20px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: gray;
	selection-background-color: rgb(191,193,194);
}

QPushButton:hover {
	background-color:rgb(105,105,105);
	border-radius: 3px;
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Insert default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pPushButtonReadFile">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	min-height: 20px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: gray;
	selection-background-color: rgb(191,193,194);
}

QPushButton:hover {
	background-color:rgb(105,105,105);
	border-radius: 3px;
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Read from File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pPushButtonCancel">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	min-height: 20px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: gray;
	selection-background-color: rgb(191,193,194);
}

QPushButton:hover {
	background-color:rgb(105,105,105);
	border-radius: 3px;
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Resources/Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
